cluster/Vagrantfile

```diff
@@ -4,32 +4,49 @@ require 'rubygems'
 require 'fileutils'
 require 'yaml'
 
-DEFAULT_K8S_VERSION = 'v1.4.7'.freeze
+DEFAULT_K8S_VERSION = 'v1.6.4'.freeze
+
+# Guest OS versions for testing
 UBUNTU = 'ubuntu'.freeze
 CENTOS = 'centos'.freeze
 RHEL = 'rhel7'.freeze
 
+HOST_SHARED_FOLDER = './export/'.freeze
+# Different orchestration platforms we support
+ORC_LEGACY_SWARM = 'docker'.freeze
+ORC_SWARM = 'swarm'.freeze
+ORC_KUBEADM = 'kubeadm'.freeze
+ORC_KUBEADM_IDX = 2
+
+# Global vars
 token = 'd900e1.8a392798f13b33a4'
 node_os = ENV['CONTIV_NODE_OS'] || CENTOS
+orc = ENV['CONTIV_ORC'] || ORC_LEGACY_SWARM
+k8s_ver = ENV['CONTIV_K8S_VERSION'] || DEFAULT_K8S_VERSION
+orc_path = k8s_ver.include?('1.5.') || k8s_ver.include?('1.4.') ? 'k8s1.4/' : 'k8s1.6/'
```

Comment:
@dseevr , I think what we have should work for 1.7. Will test and update if required.